docker/base/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:16.04
+
+MAINTAINER Beaker Feedback <beaker-feedback@twosigma.com>
+
+RUN useradd beakerx --create-home
+
+ENV JAVA_HOME /opt/conda/bin/java
```

Comment:
is this really needed?  i think the "source activate beakerx" is enough to use java from conda.